compiler/modules/front/codeGen.bal

```diff
@@ -1229,6 +1245,108 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
     panic err:impossible("unrecognized expression type in code gen: " +  s:exprToString(expr));
 }
 
+function intOperand(CodeGenContext cx, bir:Operand operand, Range subExprRange) returns bir:IntOperand|CodeGenError {
```

Comment:
Put this function down the bottom. Make the argument be `Position|Range pos`.

Reply:
Use this function in codeGenExprForInt.

Reply:
Actually better to pass in Expr rather than Range.  We don't want to create the Range in the normal case where there's no error.

Reply:
fixed